apiserver/common/networkingcommon/types.go

```diff
@@ -215,6 +218,10 @@ func networkAddressStateArgsForHWAddr(devs corenetwork.InterfaceInfos, hwAddr st
 	var res []state.LinkLayerDeviceAddress
 
 	for _, dev := range devs.GetByHardwareAddress(hwAddr) {
+		if dev.PrimaryAddress().Value == "" {
```

Comment:
Same question here; why are these filtered out?

Reply:
This serves the same function. Where we locate a device without an address, we don't attempt transformation of a zero-value address to state args.